playbooks/ansible-cloud/vcenter-appliance/files/bootstrap.yaml

```diff
@@ -14,39 +19,7 @@
         key: "{{ _ssh_key }}"
         state: present
         user: zuul
+        path: "{{ authorized_keys_file[inventory_hostname] }}"
```

Comment:
What is happening here?  Why do we need to change authorized_key file?

Reply:
On ESXi, the authorized_keys file is not in `~zuul/.ssh` directory, but in `/etc/ssh`. There is no `/home` directory.